chcemvediet/apps/inforequests/admin.py

```diff
@@ -328,6 +337,79 @@ def has_delete_permission(self, request, obj=None):
             return True
         return False
 
+    def delete_view(self, request, object_id, extra_context=None):
+        "The 'delete' admin view for this model."
+        opts = self.model._meta
+        app_label = opts.app_label
+
+        obj = self.get_object(request, unquote(object_id))
+
+        if not self.has_delete_permission(request, obj):
+            raise PermissionDenied
+
+        if obj is None:
+            raise Http404(
+                _('%(name)s object with primary key %(key)r does not exist.') %
+                {'name': force_text(opts.verbose_name), 'key': escape(object_id)}
+            )
+
+        using = router.db_for_write(self.model)
+
+        # Populate deleted_objects, a data structure of all related objects that
+        # will also be deleted.
+        (deleted_objects, perms_needed, protected) = get_deleted_objects(
+            [obj], opts, request.user, self.admin_site, using)
+        insert_inforequestemail(deleted_objects)
+
+        if request.POST:  # The user has already confirmed the deletion.
+            if perms_needed:
+                raise PermissionDenied
+            obj_display = force_text(obj)
+            self.log_deletion(request, obj, obj_display)
+            self.delete_model(request, obj)
+
+            return self.response_delete(request, obj_display)
+
+        object_name = force_text(opts.verbose_name)
+
+        if perms_needed or protected:
+            title = _("Cannot delete %(name)s") % {"name": object_name}
+        else:
+            title = _("Are you sure?")
+
+        context = dict(
+            self.admin_site.each_context(),
+            title=title,
+            object_name=object_name,
+            object=obj,
+            deleted_objects=deleted_objects,
+            perms_lacking=perms_needed,
+            protected=protected,
+            opts=opts,
+            app_label=app_label,
+            preserved_filters=self.get_preserved_filters(request),
+        )
+        context.update(extra_context or {})
+        return self.render_delete_form(request, context)
+
     def delete_model(self, request, obj):
         # delete action.email.inforequestemail_set.all()
         super(ActionAdmin, self).delete_model(request, obj)
+
+def insert_inforequestemail(deleted_objects):
```

Comment:
![Screenshot_2020-06-22 Are you sure Django site admin](https://user-images.githubusercontent.com/29918752/85316205-abf47f00-b4bc-11ea-9673-f12df76e1588.png)
@martinmacko47 Chcel som, aby sa `inforequestemail`-y vnorili pod akcie ku ktorym patri. `get_deleted_objects(` funkcia pouziva `NestedObjects` collector, ktory vsak nezahrnie dane `inforequestemail` instancie. Preto som ich tam chcel vlozit manualne iteraciou cez `deleted_objects`. Neocakaval som ze riesenie sa natolko zkomplikuje. Jednoduchsie teda bude prepisat template `delete_confirmation.html` a vsetky `inforequestemail`-y vypisat pod sebou.

Reply:
Zbytocne komplikovane. `inforequestemail` objekty staci zavesit pramo pod hlavnu action:

```
[
  u'Action: <a href="/en/admin/inforequests/action/105/">[105] Post\xfapenie \u017eiadosti</a>',
  [
    u'Branch: <a href="/en/admin/inforequests/branch/13/">13</a>',
    [...],
    u'Branch: <a href="/en/admin/inforequests/branch/14/">14</a>',
    [...],
    u'InforequestEmail: ID, ID, ID, ...',
  ]
]
```

Reply:
:ok: